src/modules/my-positions/selectors/my-positions-summary.js

```diff
@@ -15,12 +15,13 @@ export const generateOutcomePositionSummary = memoize((adjustedPosition) => {
     return null
   }
   const outcomePositions = Array.isArray(adjustedPosition) ? adjustedPosition.length : 1
-  const qtyShares = accumulate(adjustedPosition, 'numSharesAdjusted')
+  const netPosition = accumulate(adjustedPosition, 'numSharesAdjustedForUserIntention')
+  const qtyShares = accumulate(adjustedPosition, 'numShares')
   const realized = accumulate(adjustedPosition, 'realizedProfitLoss')
   const unrealized = accumulate(adjustedPosition, 'unrealizedProfitLoss')
   // todo: check if this calculation is correct for UI
   const averagePrice = accumulate(adjustedPosition, 'averagePrice')
-  const isClosable = !!createBigNumber(qtyShares || '0').toNumber() // Based on position, can we attempt to close this position
+  const isClosable = !!createBigNumber(qtyShares || netPosition || '0').toNumber() // Based on position, can we attempt to close this position
```

Comment:
I think this line needs be changed to something like:

```
const isClosable = !!((qtyShares && qtyShares.toNumber()) || (netPosition && netPosition.toNumber()))
```

I checked out this branch and went to the "What will be the #1 killer..." categorical market to test https://app.clubhouse.io/augur/story/13366/fix-close-position-for-categoricals. I then sold 0.0009998 shares of the outcome "Cancer" @	0.2801 ETH for a user who had no other positions.

In this scenario, I think the expected behavior should now be that `isClosable` is set to true. However, it is getting set to false because `qtyShares` is a BigNumber, so "qtyShares || netPosition || '0'" evaluates to `qtyShares`, which in this case is 0 (as a BigNumber).

Reply:
not the only place that needs an update, i have fixed it locally, working on debugging a metamask bug that appeared